elide-graphql/src/main/java/com/yahoo/elide/graphql/parser/GraphQLEntityProjectionMaker.java

```diff
@@ -568,13 +554,24 @@ private void addAttributeArgument(Argument argument, EntityProjectionBuilder pro
         String argumentName = argument.getName();
         Class<?> entityType = projectionBuilder.getType();
 
-        Attribute argumentAttribute = projectionBuilder.getAttributeByAlias(argumentName);
+        Attribute existingAttribute = projectionBuilder.getAttributeByAlias(argumentName);
```

Comment:
This doesn't seem right to me.  We shouldn't be fetching the attribute by the argument name.